scripts/typescript-compatibility-build.yml

```diff
@@ -120,3 +126,57 @@ jobs:
           node common/scripts/install-run-rush.js test --to $(TypeScriptVersion)-office-ui-fabric-react-test
         displayName: Build TypeScript $(TypeScriptVersion) test app
         workingDirectory: $(Build.StagingDirectory)/__tests__
+
+      # Write successful ts versions to compatMap.txt
+      - script: |
+          echo $(TypeScriptVersion) >> $(Build.StagingDirectory)/drop/compatMap.txt
+        displayName: Write successful ts versions to compatMap.txt
+
+      # Publish built library for validation job
+      - task: PublishBuildArtifacts@1
+        displayName: Publish Build Artifacts
+        inputs:
+          pathtoPublish: $(Build.StagingDirectory)/drop
+          artifactName: drop
+
+  - job: InvokePOSTAPI
+    dependsOn: Compatibility
+    condition: always()
+    timeoutInMinutes: 60
+    pool:
+      vmImage: $(VmImage)
+    steps:
+      # Download build library
+      - task: DownloadBuildArtifacts@0
+        displayName: 'Download Build Artifacts'
+        inputs:
+          artifactName: drop
+          downloadPath: $(Build.StagingDirectory)
+
+      # Invoke POST API with json body
+      - powershell: $i = 0;
```

Comment:
Does powershell work with a Linux VM Image now? Maybe it does now... but probably if we're gonna specify ubuntu, we should use bash script. Further, if we're doing something this complex, we should probably check this into a script (maybe could use node scripts at that point to be portable)

Reply:
Powershell *does* work on Ubuntu. It is intended to be cross-platform just like dotnet, and is part of our larger company direction of embracing open source and cross-plat.

I do not think there is a need to write in bash, writing in powershell ensures greater portability and also is more aligned with the rest of Microsoft.

Reply:
I agree that it would be nice to check in the script in a separate file (or at least indent this one properly if that's possible?)

Reply:
> I agree that it would be nice to check in the script in a separate file (or at least indent this one properly if that's possible?)

@ecraig12345, I have taken care of the indentation and added documentation for the InvokePOSTAPI job as per your offline comment. Triggered a build to ensure that the script works with the indentation changes.

Reply:
@kenotron @ecraig12345 There have been other comments about reviewing the design of this pipeline. About moving the REST API invocation to a separate script, can we discuss this and do it as part of the revised design ?